spec/ace.md

```diff
@@ -248,11 +248,13 @@ The App Container specification defines an HTTP-based metadata service for provi
 ### Metadata Service
 
 The ACE must provide a Metadata service on the address given to the applications via the `AC_METADATA_URL` [environment variable](#execution-environment).
+This URL must reference an endpoint which conforms to the HTTP protocol with TLS enabled except in cases where the Metadata Service resides on the same physical host.
```

Comment:
I think it's worth writing this spec from the point of the application and the environment it needs to support and expect. In that sense, we should not mandate that a runtime must use TLS; only that the `AC_METADATA_URL` may use `http` or `https` scheme and that both must be supported. If the runtime has setup an `stunnel` or uses IPSec, it may not want to use TLS at the application level.